plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -89,17 +116,19 @@ def __init__(self, ufm_client: UFMCommunicator, logger):
         self.deisolate_consider_time = pdr_config.getint(Constants.CONF_COMMON,Constants.DEISOLATE_CONSIDER_TIME)
         self.automatic_deisolate = pdr_config.getboolean(Constants.CONF_COMMON,Constants.AUTOMATIC_DEISOLATE)
         self.dynamic_wait_time = pdr_config.getint(Constants.CONF_COMMON,"DYNAMIC_WAIT_TIME")
+        self.temp_check = pdr_config.getboolean(Constants.CONF_COMMON,Constants.CONFIGURED_TEMP_CHECK)
+        self.no_down_count = pdr_config.getboolean(Constants.CONF_COMMON,Constants.NO_DOWN_COUNT)
+        self.access_isolation = pdr_config.getboolean(Constants.CONF_COMMON,Constants.ACCESS_ISOLATION)
         # Take from Conf
         self.logger = logger
-        self.isolation_matrix = pd.read_csv(Constants.BER_MATRIX)
-        min_threshold = self.isolation_matrix[[
-            Constants.CSV_RAW_BER_ISOLATE, Constants.CSV_RAW_BER_DEISOLATE,
-            Constants.CSV_EFF_BER_ISOLATE, Constants.CSV_EFF_BER_DEISOLATE, 
-            Constants.CSV_SYMBOL_BER_ISOLATE, Constants.CSV_SYMBOL_BER_DEISOLATE]].min().min()
-        self.ber_wait_time = self.calc_max_ber_wait_time(min_threshold)
+        intervals = [x[0] for x in Constants.BER_THRESHOLDS_INTERVALS]
+        self.min_ber_wait_time = min(intervals)
+        self.max_ber_wait_time = max(intervals)
+        self.max_ber_threshold = max([x[1] for x in Constants.BER_THRESHOLDS_INTERVALS])
```

Comment:
IMHO - no need to keep pairs of intervals if you do not use them as pairs

Reply:
each interval is bundled with its own threshold to check as given from the algorithm so that's why they're tuples